ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/PivotSelectorFromSafeBlock.java

```diff
@@ -69,6 +71,10 @@ public Optional<FastSyncState> selectNewPivotBlock() {
     if (maybeForkchoice.isPresent() && maybeForkchoice.get().hasValidSafeBlockHash()) {
       return Optional.of(selectLastSafeBlockAsPivot(maybeForkchoice.get().getSafeBlockHash()));
     }
+    if (lastNoFcuReceivedInfoLog + NO_FCU_RECEIVED_LOGGING_THRESHOLD < System.currentTimeMillis()) {
+      lastNoFcuReceivedInfoLog = System.currentTimeMillis();
+      LOG.info("Waiting on CL to supply a pivot block, please check if your CL is in sync");
```

Comment:
I'm not sure most user understand what "CL" means

Reply:
What do you reckon is a better message?

Reply:
`Waiting for consensus client, this may be because your consensus is still syncing`

CL and pivot block are not generally understood. We also don't want to suggest there's an ongoing problem, waiting is expected.